pkg/controller/control.go

```diff
@@ -21,14 +21,16 @@ import (
 
 type Controller struct {
 	sync.RWMutex
-	Name       string
-	size       int64
-	sectorSize int64
-	replicas   []types.Replica
-	factory    types.BackendFactory
-	backend    *replicator
-	frontend   types.Frontend
-	isUpgrade  bool
+	Name                      string
+	size                      int64
+	sectorSize                int64
+	replicas                  []types.Replica
+	factory                   types.BackendFactory
+	backend                   *replicator
+	frontend                  types.Frontend
+	isUpgrade                 bool
+	iscsiTargetRequestTimeout int64
+	engineReplicaTimeout      time.Duration
```

Comment:
Can we use `time.Duration` for all timeout properties to make type consistent? You can use `.Seconds()` to get the seconds later.

Reply:
Sure.

Reply:
Addressed. Thank you.